zkstack_cli/crates/zkstack/src/commands/chain/gateway/gateway_common.rs

```diff
@@ -72,6 +72,8 @@ pub enum GatewayMigrationProgressState {
     PendingManualFinalization,
     /// The migration has finished.
     Finished,
+    /// The migration has started, but Gateway and L1 shows different settlement layers
+    UndefinedSettlementLayer,
```

Comment:
this state is provided against some migration direction, so it can not be undefined